cli/config/config.go

```diff
@@ -39,7 +40,7 @@ func (c *Config) Marshal() []byte {
 	return buf.Bytes()
 }
 
-func (c *Config) Add(s *Cluster) error {
+func (c *Config) Add(s *Cluster, f bool, d bool) error {
```

Comment:
Even though Golang usually prefers short variable names, I'd use `force` and `default` here, so it's a bit more obvious what's going on (`s` is a bit more straightforward since the type tells you what it is).